### ISSUE TITLE
Clean up document with clear structure

### ISSUE THREAD

**Issue by simnh** [simnh/addressing_challenges_oemof#5]
I think it is time to clean up the document, as there has been added a lot of text/content and literature so far. I will use this week to clean up the document and provide a preliminary finalized structure for the paper. 

This will include: 
- Structure the subsections 
- Create text from bullitpoints
- Remove duplicate content (merge/remove) 
- Clean up todos (remove old ones, create new small clear ones)

If you have any new additional input feel free to add it under ideas.tex with a small comment where it should be inserted.

**Comment by gplssm** [comment]
Great initiative at the right moment!

I still have some uncommitted/unpushed content on my disc for Subsection `uncertainties` to get this subsection to a final draft state. I'll try try to finalize and publish today.
Furthermore, the subsection `inaccuracy` is still not finished: more stringent text and references are needed. I'm not sure if I can make it this week.

Besides that, I'd like to suggest a slightly different process
- Structure the subsections, rethink the overall storyline based on current and expected future content
- Remove duplicate content (merge/remove) 
- Assign Sections/Subsections to co-authors that they create text from bulletpoints (incl. deadline)
- Clean up todos (remove old ones, create new small clear ones, assign them)

**Comment by simnh** [comment]
Ok, i already pushed some restructured text (no rst-files ;) ), so you may run into trouble. Maybe it's better not to merge but to just copy paster your additional ideas to the newset master commit.

**Comment by gplssm** [comment]
Thanks for your work - it's great! I realized that I already added all relevant stuff and you put it into real text. :)

**Comment by fraukewiese** [comment]
Thanks a lot for cleaning up, Simon, already quite well structured I think.

I have tried to finish the introduction, added some comments on the oemof description and made some suggestion for contribution (see pull request). I added some discussion points in the discussion section, but don't know if this matches your idea of what should be in there.  So feel free to add, comment, throw out.
I could take some more hours next Friday to contribute to the paper. Thus if you have specific points where you think you'd like to get feedback to written stuff or figures, or sections I should write or improve, just tell me.
